.github/workflows/motion-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Motion Extended Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  check-files:
+    name: Check files
+    runs-on: ubuntu-latest
+    outputs:
+      EXTENDED_MOTION_TEST: ${{ steps.file_checker.outputs.EXTENDED_MOTION_TEST }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.sha || github.ref }}
+          fetch-depth: 0
+      - name: check modified files
+        id: file_checker
+        run: |
+          ( git diff --name-only origin/${{github.base_ref}} | grep -e "motionplan" \
```

Comment:
can potentially use the built in paths filter for this

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

and could do it in test.yaml where you call down -- that would let you remove the conditional logic from this file + make it easier to run manually as you test

would also need to change `result }}" == "success` at the bottom of test.yml to include the skipped case

Reply:
I believe that will only function for triggering a workflow, right? Not for running a workflow triggered elsewhere? This makes it hard to have the step be required